packages/catalog-process/src/utilities/errorMappers.ts

```diff
@@ -49,6 +49,7 @@ export const documentCreateErrorMapper = (
       () => HTTP_STATUS_NOT_FOUND
     )
     .with("notValidDescriptor", () => HTTP_STATUS_BAD_REQUEST)
+    .with("interfaceAlreadyExists", () => HTTP_STATUS_CONFLICT)
```

Comment:
```suggestion
    .with("interfaceAlreadyExists", () => HTTP_STATUS_BAD_REQUEST)
```
In scala this is `badRequest`, let's keep the same logic

Reply:
Sure, updated.